### PR TITLE
Register variables for customisation.

### PR DESCRIPTION
**Pull request by bierik**
Fixes https://github.com/4teamwork/plonetheme.blueberry/issues/2

bildschirmfoto 2016-02-09 um 14 44 53

### CODE REVIEW

**Comment by jone** [comment]
The tests are not passing
```
FormFieldNotFound: Could not find form field: "$secondary-color". Fields: "Search Site", "only in current section", "$color-primary This color is the main color of the page. It appears the most on the page. Should be the main color of the CI/CD. This color is also used for links, buttons and tabs.", "$color-secondary This color is used widley in the page but less than the primary color. Should be the one of the main colors of the CI/CD. This color is also used for links and buttons.", "$color-edit This color is only visible in the logged in state. It is used to indicate edit sections e.q. Simplelayout or the portal top navigation", "$color-link This color is used for the default state of links. The default color is $color-primary", "$color-link-hover This color is used for the hover state of links. The default color is $color-secondary.", "$font-size-base Base font size.", "$line-height-base Base line height.", "form.buttons.save", "form.buttons.cancel"
```
looks like the html markup has changed?

File: plonetheme/blueberry/locales/de/LC_MESSAGES/plonetheme.blueberry.po [hunk 1]
```diff
@@ -46,183 +51,48 @@ msgstr "Weitere Informationen."
...
-msgstr "Farbe für Knöpfe, welche eine gefährliche Aktion auslösen."
-
-#. Default: "Color of buttons causing a destructive action, e.g. deleting content."
-#: ./plonetheme/blueberry/browser/forms.py:79
-msgid "label_button_destructive_color"
-msgstr "Farbe für Knöpfe, welche eine destruktive Aktion auslösen, z.B. Inhalt löschen."
+#. Default: "This color is only visible in the logged in state. It is used to indicate edit sections e.q. Simplelayout or the portal top navigation"
+#: ./plonetheme/blueberry/browser/forms.py:54
+msgid "label_color_edit"
+msgstr "Diese Farbe wird nur im eingeloggten Zustand sichtbar. Sie wird gebraucht um Bereichte, die zum bearbeiten von Inhalten gedacht sind, zu markieren wie z.B. im Simplelayout oder in der top Navigation."
```
Comment (jone) [inline plonetheme/blueberry/locales/de/LC_MESSAGES/plonetheme.blueberry.po]: `Bereichte` » `Bereiche`
`top Navigation` » `Hauptnavigation`

File: plonetheme/blueberry/locales/de/LC_MESSAGES/plonetheme.blueberry.po [hunk 2]
```diff
@@ -46,183 +51,48 @@ msgstr "Weitere Informationen."
...
-msgstr "Farbe für Knöpfe, welche erfolgreiche Aktion auslösen, z.B. Speichern eines Dokuments."
-
-#. Default: "A dark color for buttons with light colors (applied automatically for maximum contrast)."
-#: ./plonetheme/blueberry/browser/forms.py:103
-msgid "label_button_text_dark_color"
-msgstr "Eine dunkle Farbe für das Label von Knöpfen mit heller Hintergrundfarbe."
+#. Default: "This color is used for the default state of links. The default color is $color-primary"
+#: ./plonetheme/blueberry/browser/forms.py:59
+msgid "label_color_link"
+msgstr "Diese Farbe wird für den initial Zustand von Links verwendet. Die Standardfarbe ist $color-primary"
```
Comment (jone) [inline plonetheme/blueberry/locales/de/LC_MESSAGES/plonetheme.blueberry.po]: `initial Zustand` » `Initialzustand`

**Comment by jone** [comment]
:+1: Awesome

**Comment by jone** [comment]
@bierik please close this one and make a new PR with target `pu`, as discussed :wink:

**Comment by bierik** [comment]
Moved to https://github.com/4teamwork/plonetheme.blueberry/pull/19

**Pull request closed without merging**